bundles/org.openhab.binding.mqtt.espmilighthub/src/main/resources/OH-INF/i18n/mqtt.properties

```diff
@@ -22,31 +22,29 @@ thing-type.config.mqtt.cct.oneTriggersNightMode.description = 1% on a slider wil
 thing-type.config.mqtt.rgbandcct.favouriteWhite.label = Favourite White
 thing-type.config.mqtt.rgbandcct.favouriteWhite.description = When a shortcut triggers white mode, use this for the colour white.
 thing-type.config.mqtt.rgbandcct.oneTriggersNightMode.label = 1% Triggers Night Mode
 thing-type.config.mqtt.rgbandcct.oneTriggersNightMode.description = 1% on a slider will trigger the Night Mode.
 thing-type.config.mqtt.rgbandcct.powerFailsToMinimum.label = Dimmed on Power Fail
-thing-type.config.mqtt.rgbandcct.powerFailsToMinimum.description = If lights loose power, the lights will turn on to the minimum brightness.
+thing-type.config.mqtt.rgbandcct.powerFailsToMinimum.description = If lights lose power, the lights will turn on to the minimum brightness.
 thing-type.config.mqtt.rgbandcct.whiteHue.label = White Hue
 thing-type.config.mqtt.rgbandcct.whiteHue.description = When both the whiteHue and whiteSat values are seen by the binding it will trigger the white LEDS.
 thing-type.config.mqtt.rgbandcct.whiteSat.label = White Saturation
 thing-type.config.mqtt.rgbandcct.whiteSat.description = When both the whiteHue and whiteSat values are seen by the binding it will trigger the white LEDS.
-thing-type.config.mqtt.rgbandcct.whiteThreshold.label = White Threshold
-thing-type.config.mqtt.rgbandcct.whiteThreshold.description = Saturation values at or below this value on a RGBW color control will trigger the white mode. -1 will disable this feature.
 thing-type.config.mqtt.rgbw.oneTriggersNightMode.label = 1% Triggers Night Mode
 thing-type.config.mqtt.rgbw.oneTriggersNightMode.description = 1% on a slider will trigger the Night Mode.
 thing-type.config.mqtt.rgbw.powerFailsToMinimum.label = Dimmed on Power Fail
-thing-type.config.mqtt.rgbw.powerFailsToMinimum.description = If lights loose power, the lights will turn on to the minimum brightness.
+thing-type.config.mqtt.rgbw.powerFailsToMinimum.description = If lights lose power, the lights will turn on to the minimum brightness.
 thing-type.config.mqtt.rgbw.whiteHue.label = White Hue
 thing-type.config.mqtt.rgbw.whiteHue.description = When both the whiteHue and whiteSat values are seen by the binding it will trigger the white LEDS.
 thing-type.config.mqtt.rgbw.whiteSat.label = White Saturation
 thing-type.config.mqtt.rgbw.whiteSat.description = When both the whiteHue and whiteSat values are seen by the binding it will trigger the white LEDS.
-thing-type.config.mqtt.rgbw.whiteThreshold.label = White Threshold
-thing-type.config.mqtt.rgbw.whiteThreshold.description = Saturation values at or below this value on a RGBW color control will trigger the white mode. -1 will disable this feature.
```

Comment:
This one must not be removed, whiteThreshold i still there for rgbw.
The one to be suppressed is for rgbandcct.